src/ip-fragment.c

```diff
@@ -162,18 +212,41 @@ static int ip_fragment_init(struct pg_brick *brick,
 		return -1;
 	}
 
+	if (ip_fragment_config->mtu_size % 8) {
+		*errp = pg_error_new("mtu must be a  multiplier");
+		return -1;
+	}
 	brick->burst = ip_fragment_burst;
 	state->output = ip_fragment_config->output;
 	state->mtu_size = ip_fragment_config->mtu_size;
+	state->tbl = rte_ip_frag_table_create((1 << 15), 4,
+					      1 * ((1 << 15)),
+					      /* ~3s of timeout per segment */
+					      rte_get_timer_hz() * 3,
+					      SOCKET_ID_ANY);
```

Comment:
`rte_get_timer_hz` return the number of cycle by seconde `* 3` make a multiplication by 3...